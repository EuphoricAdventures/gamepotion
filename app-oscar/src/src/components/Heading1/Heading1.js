import React from 'react'
import styled from 'styled-components'
import { font, colours } from '../../styleAbstractions'

const StyledHeading1 = styled.h1`
  ${font}
  font-size: 200%;
  font-weight: 800;
  color: ${colours.fore};
  // background-color: red;
`

const Heading1 = ({ children }) => {
  return (
    <StyledHeading1 className='component--heading1'>
      {children}
    </StyledHeading1>
  )
}

export default Heading1
