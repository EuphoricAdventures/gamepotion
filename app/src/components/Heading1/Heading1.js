import React from 'react'
import styled from 'styled-components'
import { font, colours } from '../abstractions'

const StyledHeading1 = styled.h1`
  display: block;
  ${font}
  font-size: 200%;
  font-weight: 800;
  color: ${colours.fore};
  // background-color: red;
`

const Heading1 = ({ children }) => {
  return (
    <StyledHeading1 className='component--heading1'>
      {children}
    </StyledHeading1>
  )
}

export default Heading1
