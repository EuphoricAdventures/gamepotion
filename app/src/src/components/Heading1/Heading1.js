import React from 'react'
import styled from 'styled-components'

const styleAbstractions = {
  colours: {
    'fore': '#2e3131'
  },
  fontAlternative: `
    font-family: "Rubik", -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
    font-weight: 400;
    text-size-adjust: 100%;
    -webkit-font-smoothing: antialiased;
  `
}

const StyledHeading1 = styled.h1`
  ${styleAbstractions.fontAlternative}
  font-size: 200%;
  color: ${styleAbstractions.colours.fore};
  // background-color: red;
`

const Heading1 = ({ children }) => {
  return (
    <StyledHeading1 className='component--heading1'>
      {children}
    </StyledHeading1>
  )
}

export default Heading1
