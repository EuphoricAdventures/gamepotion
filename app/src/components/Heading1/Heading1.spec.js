import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Heading1 from './Heading1'

it('renders', () => {
  const tree = renderer
    .create(
      <Heading1>Hello, World</Heading1>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
