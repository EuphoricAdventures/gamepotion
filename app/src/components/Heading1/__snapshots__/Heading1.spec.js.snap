// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 {
  display: block;
  font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",sans-serif;
  font-weight: 400;
  -webkit-text-size-adjust: 100%;
  text-size-adjust: 100%;
  -webkit-font-smoothing: antialiased;
  font-size: 200%;
  font-weight: 800;
  color: #2e3131;
}

<h1
  className="component--heading1 c0"
>
  Hello, World
</h1>
`;
